morecantile/models.py

```diff
@@ -713,6 +716,11 @@ def custom(
             transform = pyproj.Transformer.from_crs(extent_crs, crs, always_xy=True)
             extent = transform.transform_bounds(*extent, densify_pts=21)
 
+        if decimation_base < 2:
```

Comment:
in the docstring we say that it should be greater than `1` but here we raise an error if < 2 🤷

Reply:
updated check to raise error when `<= 1`